src/controllers/postsRoute/addComment.js

```diff
@@ -0,0 +1,36 @@
+const { ObjectId } = require("mongodb");
+const { db } = require("../../database/database");
+const { validateComment } = require("../validations/validateComment");
+
+exports.addComment = async function (request, response) {
+  try {
+    const { username } = request.loggedInUser;
+
+    const validatedBody = validateComment(request.body);
+
+    if (validatedBody.error) {
+      response.status(400).send(validatedBody.error.details[0].message);
+      return;
+    }
+
+    const { commentText, postID } = validatedBody.value;
+
+    const result = await db.posts.updateOne(
+      { _id: new ObjectId(postID) },
+      {
+        $push: {
+          comments: {
+            username: username,
+            comment: commentText,
+          },
+        },
+      }
+    );
+
+    response.status(200).json(result);
```

Comment:
Jag tror att det är bra om du lägger till validering här. Jag får status 200 trots att jag skriver in ett felaktigt id som inte finns. typ return 404 om en post inte hittas med if(result.matchedCount === 0)